demos/gaze_estimation_demo/cpp/README.md

```diff
@@ -1,6 +1,6 @@
 # Gaze Estimation Demo
 
-![](./gaze_estimation.gif)
+![](../cpp_common/gaze_estimation.gif)
```

Comment:
Please, leave the *.gif in `demos/gaze_estimation_demo` directory (see [here](https://github.com/openvinotoolkit/open_model_zoo/tree/develop/demos/object_detection_demo) for example).

Reply:
Fixed. What about possible future python (for example) samples. Should these have the same results (the same accuracy, rendering, etc)?

Reply:
Yes, the should (as much as possible).